### PR TITLE
update to stable version of sbt-spiewak

### PR DESCRIPTION
**Pull request by larsrh**

### CODE REVIEW

**Comment by larsrh** [comment]
(ノಠ益ಠ)ノ彡┻━┻

**Comment by larsrh** [comment]
@djspiewak Please check if these fatal warnings in the tests are intended.

**Comment by djspiewak** [comment]
Fatal warnings are indeed intended. They weren't working before because of a bug in the plugin, which is how this code ended up in master. Leave it open; I'll fix it since it's my fault. :-)

File: build.sbt [hunk 1]
```diff
@@ -142,6 +142,7 @@ lazy val docs = project
     ScalaUnidoc / unidoc / unidocProjectFilter := inProjects(coreJVM, lawsJVM),
     docsMappingsAPIDir := "api",
     addMappingsToSiteDir(ScalaUnidoc / packageDoc / mappings, docsMappingsAPIDir),
+    scalacOptions := scalacOptions.value.filterNot(_ == "-Werror"),
```
Comment (larsrh) [inline build.sbt:145]: I attempted to disable fatal warnings in mdoc. It worked locally, but not in CI.

**Comment by larsrh** [comment]
ᕙ(⇀‸↼‶)ᕗ

**Pull request merged**